CoEpi/ble/covidwatch/BluetoothController.swift

```diff
@@ -526,8 +526,8 @@ extension BluetoothController: CBCentralManagerDelegate {
             else {
                 if (isConnectable) {
                     self.peripheralsToWriteContactEventIdentifierTo.insert(peripheral)
-                    // OR use reading. Both cases are handled.
-                    // self.peripheralsToReadContactEventIdentifierFrom.insert(peripheral)
+                    // OR use reading. Both cases are handled. - duskoo: Pretty sure we need to enable reading
+                    self.peripheralsToReadContactEventIdentifierFrom.insert(peripheral)
```

Comment:
This is a bit weird. According to https://docs.google.com/document/d/1f65V3PI214-uYfZLUZtm55kdVwoazIMqGJrxcYNI4eg/edit#heading=h.6q40wl39kcs8 at least, iOS to iOS should use characteristic read. And anyway if both devices write, they should see each other. But the timing here may not be right.

Reply:
(I assumed this fixes the issue I'm having with devices not seeing each other, but realized that it's not necessarily related. Will test later).